pages/index.vue

```diff
@@ -1,6 +1,19 @@
 <template>
   <div class="page">
-    <pw-section class="cyan" label="Request" ref="request">
+    <pw-section class="blue" label="Request" ref="request">
+      <button id="show-modal" @click="showModal = true">IMPORT</button>
```

Comment:
Can you rename this to 'Import' and move it outside of the `pw-section` for request?

Reply:
@NBTX  I have made the requested changes, please have a look.